test/v1/validation/DeemedParentValidatorSpec.scala

```diff
@@ -165,44 +171,37 @@ class DeemedParentValidatorSpec extends BaseSpec {
 
         "IsUK and CountryOfIncorporation is supplied" in {
 
-          val model = deemedParentModelNonUkCompany.copy(
-            isUk = true
-          )
-
+          val model = deemedParentModelNonUkCompany.copy(isUk = true)
           leftSideError(model.validate).errorMessage shouldBe WrongDeemedParentIsUkPartnershipAndNonUKCompany(model).errorMessage
         }
 
-        "NonUK and SAUTR is supplied" in {
-
-          val model = deemedParentModelUkCompany.copy(
-            isUk = false
-          )
+        "NonUK and SAUTR is supplied by default" in {
 
+          val model = deemedParentModelUkCompany.copy(isUk = false)
           leftSideError(model.validate).errorMessage shouldBe WrongDeemedParentIsUkPartnershipAndNonUKCompany(model).errorMessage
         }
 
-        "NonUK and CTUTR is supplied" in {
+        "NonUK and No SAUTR is supplied explicitly" in {
+
+          val model = deemedParentModelNonUkCompany.copy(isUk = false, sautr = Some(UTRModel("1123456789")))
+          leftSideError(model.validate).errorMessage shouldBe WrongDeemedParentIsUkPartnershipAndNonUKCompany(model).errorMessage
+        }
```

Comment:
The above two tests are basically the same - deemedParentModelUkCompany is just a test asset that has been coded to contain an SAUTR. If you copy it and give it the same sautr it's still basically the same object. Is this what you were intending to test?

Same below.

Reply:
Yes. There is an assumption being made that deemedParentModelNonUkCompany will be correct in the first test, but not in the second one. If one fails, but not the other that's a huge indicator where the problem is.

Reply:
Do we just want to have the second test then?